packages/thumbprint-atomic/src/packages/flexbox.scss

```diff
@@ -205,6 +205,78 @@
     flex-shrink: 1 !important;
 }
 
+.col-gap0 {
```

Comment:
This makes sense in its own file and documentation section, IMO, since these gap classes work for both flexbox or CSS grid. (I don't think there's much CSS grid use in our codebase, but it is production ready and should be added to Atomic eventually.)

Reply:
Silly question, but can we leverage scss for loops to generate incremental class structures like this?

Reply:
Moved it to its own file.

Yeah it's quite error prone doing it via copy-paste so I think using scss loops would be good. But yeah, we'd probably want to do it across the atomic codebase.